bad_connector_woocommerce/models/woo_settings/importer.py

```diff
@@ -67,4 +67,44 @@ def _after_import(self, binding, **kwargs):
             else:
                 include_tax = False
             binding.backend_id.write({"include_tax": include_tax})
+
+        if binding.external_id == "woocommerce_currency":
+            currency = self.env["res.currency"].search(
+                [("name", "=", binding.value)], limit=1
```

Comment:
best will be based on code.

Reply:
The name field in res.currency actually consist the code of currency.